src/main/webapp/WEB-INF/web.xml

```diff
@@ -0,0 +1,24 @@
+<web-app xmlns="http://xmlns.jcp.org/xml/ns/javaee"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee
+         http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd
+         http://xmlns.jcp.org/xml/ns/javaee "
+         version="3.1">
+
+    <welcome-file-list>
+        <welcome-file>index.html</welcome-file>
+    </welcome-file-list>
+
+
+    <servlet>
+        <display-name>DictionaryServlet</display-name>
+        <servlet-name>DictionaryServlet</servlet-name>
+        <servlet-class>src/main/java/munitz/scrabble/DictionaryServlet.java</servlet-class>
```

Comment:
[This is not the correct servlet-clas](https://docs.oracle.com/cd/E13222_01/wls/docs92/webapp/configureservlet.html#wp120130)

Reply:
Okay,  I fixed that but now I get the following error: `java.io.FileNotFoundException: src\main\java\munitz\scrabble\dictionary.txt (The system cannot find the path specified)`
Do I need to move the txt file somewhere else?